src/datajudge/constraints/base.py

```diff
@@ -119,10 +120,7 @@ def __init__(
         ref2=None,
         ref_value: Any = None,
         name: str = None,
-        output_processor: Callable[
-            [Collection, Optional[Collection]], Collection
-        ] = None,
-        output_remainder_slicer=slice(5),
+        output_processors: List[OutputProcessor] = None,
```

Comment:
As discussed you could define this as `OutputProcessor | List[OutputProcessor]` and in the body you can do something like

```python
if not isinstance(output_processor, List]:
    output_processor = [output_processor]
```

Reply:
Done